types/options.pp

```diff
@@ -9,4 +9,6 @@
     Optional[maxelem]  => Integer[128],
     Optional[netmask]  => IP::Address,
     Optional[timeout]  => Integer[1],
+    Optional[range]    => String,
+    Optional[comment]  => String,
```

Comment:
can you make this a bit more strict? Are empty strings okay or should we enforce a minimal length?

Reply:
```puppet
Optional[range]    => Pattern[/^6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}-6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}$/],
Optional[comment]  => Pattern[/^(.*").*$/],
```
Port regex blatantly stolen from [StackOverflow](https://stackoverflow.com/questions/3715064/anyone-see-anything-wrong-with-my-regex-for-port-numbers) 

According do [documentation](https://ipset.netfilter.org/ipset.man.html#lbAL) only `"` is an illegal character in the comment.